tests/pdctl/keyspace/keyspace_group_test.go

```diff
@@ -95,8 +95,12 @@ func TestSplitKeyspaceGroup(t *testing.T) {
 	defer cancel()
 	re.NoError(failpoint.Enable("github.com/tikv/pd/pkg/keyspace/acceleratedAllocNodes", `return(true)`))
 	re.NoError(failpoint.Enable("github.com/tikv/pd/server/delayStartServerLoop", `return(true)`))
+	keyspaces := make([]string, 0)
+	for i := 0; i < 500; i++ {
```

Comment:
Why 500? I think a little bit large than 128 is enough.

Reply:
sure